python/ray/scripts/scripts.py

```diff
@@ -706,6 +706,17 @@ def get_worker_ips(cluster_config_file, cluster_name):
     click.echo("\n".join(worker_ips))
 
 
+@cli.command()
+@click.argument("command", required=True, type=str)
+@click.option(
+    "--dry",
+    is_flag=True,
+    default=False,
+    help="Print actions instead of running them.")
+def session(command, dry):
+    ray.projects.load_project(os.getcwd())
```

Comment:
I'm not sure if I understand this command...this is eventually how a session should be started, right? What is it supposed to do right now?

Reply:
Right now it just validates the project file. @simon-mo will start working on the implementation that starts the session soon.